daemons/fenced/fenced_remote.c

```diff
@@ -72,18 +72,19 @@ typedef struct st_query_result_s {
     int ndevices;
     /* Devices available to this host that are capable of fencing the target */
     GHashTable *devices;
-} st_query_result_t;
+} peer_device_info_t;
```

Comment:
any reason why we don't alter 'struct st_query_result_s' consistently?

Reply:
no, missed that

Reply:
added commit on top for this and the other rename